my-app/src/categories.json

```diff
@@ -0,0 +1,10 @@
+[{
```

Comment:
Move all the json files to `mocks` folder.

You can then use this data as fallback if the API call to server fails